src/surface.cpp

```diff
@@ -36,16 +36,16 @@ using namespace std;
 // RGBAColor:
 
 RGBAColor RGBAColor::fromString(const string &strColor) {
-	return {
-		uint8_t(std::clamp(std::stoi(strColor.substr(0, 2), nullptr, 16),
-		                  0, 255)),
-		uint8_t(std::clamp(std::stoi(strColor.substr(2, 2).c_str(), nullptr, 16),
-		                  0, 255)),
-		uint8_t(std::clamp(std::stoi(strColor.substr(4, 2).c_str(), nullptr, 16),
-		                  0, 255)),
-		uint8_t(std::clamp(std::stoi(strColor.substr(6, 2).c_str(), nullptr, 16),
-		                  0, 255)),
-	};
+  return {
+      uint8_t(
+          compat::clamp(std::stoi(strColor.substr(0, 2), nullptr, 16), 0, 255)),
+      uint8_t(compat::clamp(
+          std::stoi(strColor.substr(2, 2).c_str(), nullptr, 16), 0, 255)),
+      uint8_t(compat::clamp(
+          std::stoi(strColor.substr(4, 2).c_str(), nullptr, 16), 0, 255)),
+      uint8_t(compat::clamp(
+          std::stoi(strColor.substr(6, 2).c_str(), nullptr, 16), 0, 255)),
+  };
```

Comment:
You changed tabs to spaces, please use the same code style.

Reply:
I've added a `.clang-format` file in #2 to avoid accidents like this.

Reply:
Fixed